src/app/settings/views/Configuration/GeneralForm/_index.scss

```diff
@@ -0,0 +1,6 @@
+@mixin GeneralForm {
+    .themeRow {
```

Comment:
Could you update all class names to follow the naming convention, please. https://webteam.canonical.com/practices/css